### ISSUE TITLE
New CLI flag --browser

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#530]
Enable the user to spawn a specific browser when running from the command line.

`cypress run --browser chrome`

These videos will not be recoded and we should mention that.

**Comment by bahmutov** [comment]
Need to make sure the error to find the browser looks nice to the user. Right now has the stack, and is giant
```
  cypress:cli spawning Cypress /Users/irinakous/git/cypress-monorepo/cli/dist/Cypress.app/Contents/MacOS/Cypress +8ms
  cypress:cli args ["--project","/Users/irinakous/git/cypress-monorepo/cli","--cli-version","0.19.2","--browser","chrome"] +0ms
  cypress:cli some of the options {"verify":false,"detached":false} +0ms
  (Tests Starting)


  (All Done)

Browser: 'chrome' not found. Available browsers are: []
Error: Browser: 'chrome' not found. Available browsers are: []
    at browserNotFoundErr (/Users/irinakous/git/cypress-monorepo/cli/dist/Cypress.app/Contents/Resources/app/node_modules/@cypress/core-launcher/lib/browsers.coffee:11:11)
    at Object.launch (/Users/irinakous/git/cypress-monorepo/cli/dist/Cypress.app/Contents/Resources/app/node_modules/@cypress/core-launcher/lib/browsers.coffee:26:15)
    at Object.launch (/Users/irinakous/git/cypress-monorepo/cli/dist/Cypress.app/Contents/Resources/app/node_modules/@cypress/core-launcher/lib/launcher.coffee:20:25)
```

**Comment by brian-mann** [comment]
Yah this isn't a publicly available flag, so there needs to be code added in `lib/modes/headless` to account for this and throw a nice error in `lib/errors`

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
